### PR TITLE
[Ready] PDA hand Headset nodes!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Adds in most PDA cards into Rnd tech webs and such!
Adds in many headset keys for Rnd tech webs and such
3 nodes for PDAs 
2 nodes for Headsets

[why]
Most PDAs cards are not used at all! Maybe mainstreaming them will see more use.
Head set keys often get lost/stolen/or all gone do to reasons. Thus this new tech web node set will fix that!

### CODE REVIEW

**Comment by Trilbyspaceclone** [comment]
Triv rethink*

**Comment by Trilbyspaceclone** [comment]
REEEEE

File: code/modules/research/designs/misc_designs.dm [hunk 1]
```diff
@@ -408,3 +408,301 @@
 	build_path = /obj/item/reactive_armour_shell
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
+
+/////////////////////////////////////////
+////////////Headset Keys/////////////////
+/////////////////////////////////////////
+
+/datum/design/headset_sci
+	name = "Sci cription Key"
```
Comment (nik707) [inline code/modules/research/designs/misc_designs.dm]: All mentions of "cription" to "encryption"

File: code/modules/research/designs/misc_designs.dm [hunk 2]
```diff
@@ -408,3 +408,301 @@
...
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
+
+/////////////////////////////////////////
+////////////Headset Keys/////////////////
+/////////////////////////////////////////
+
+/datum/design/headset_sci
+	name = "Sci cription Key"
+	desc = "A headset key for sci channle."
```
Comment (nik707) [inline code/modules/research/designs/misc_designs.dm]: all mentions of "channle" to "channel"

File: code/modules/research/designs/misc_designs.dm [hunk 3]
```diff
@@ -408,3 +408,301 @@
...
+	name = "Mine cription Key"
+	desc = "A headset key that has both Sci and Cargo channles."
+	id = "headset_mining"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1500, MAT_PLASTIC = 1500, MAT_SILVER = 2500, MAT_GOLD = 2500)
+	build_path = /obj/item/encryptionkey/headset_mining
+	category = list("Equipment")
+	departmental_flags = DEPARTMENTAL_FLAG_CARGO | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/headset_rob
```
Comment (nik707) [inline code/modules/research/designs/misc_designs.dm]: Tbh a robotic encryption key being printable seems uhhhhhhhhhhhhhhhhhhhhhhhhh nuts

File: code/modules/research/techweb/all_nodes.dm [hunk 4]
```diff
@@ -135,6 +135,46 @@
...
+	display_name = "PDA Tech"
+	description = "Data chips to place in PDAs for more functions."
+	prereq_ids = list("datatheory", "telecomms")
+	design_ids = list("cart_eng", "cart_med", "cart_sci", "cart_sec", "cart_jan", "cart_cura", "cart_law", "cart_basic")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+
+/datum/techweb_node/adv_pda
+	id = "adv_pda"
+	display_name = "Advanced PDA Tech"
+	description = "Data chips to place in PDAs for more functions. These have even more fuctions then before!"
```
Comment (nik707) [inline code/modules/research/techweb/all_nodes.dm]: then to than

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -135,6 +135,46 @@
...
+	prereq_ids = list("adv_engi", "pda")
+	design_ids = list("cart_chem", "cart_atmos", "cart_dect", "cart_robo")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+
+/datum/techweb_node/deluxe_pda
+	id = "deluxe_pda"
+	display_name = "DELUXE PDA Tech"
+	description = "DELUXE Data chips to place in PDAs for more functions. Ment for Heads and higher ranking crew."
```
Comment (nik707) [inline code/modules/research/techweb/all_nodes.dm]: Ment to Meant

File: code/modules/research/techweb/all_nodes.dm [hunk 6]
```diff
@@ -135,6 +135,46 @@
...
+	prereq_ids = list("emp_adv", "adv_pda")
+	design_ids = list("cart_head", "cart_hos", "cart_cmo", "cart_hop", "cart_ce", "cart_qm", "cart_rd")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 7500)
+
+/datum/techweb_node/headset_keys
+	id = "headset_keys"
+	display_name = "Headset Keys"
+	description = "Privet channles that people use."
```
Comment (nik707) [inline code/modules/research/techweb/all_nodes.dm]: Private channels

File: code/modules/research/techweb/all_nodes.dm [hunk 7]
```diff
@@ -135,6 +135,46 @@
...
+	prereq_ids = list("adv_engi", "telecomms")
+	design_ids = list("headset_sec", "headset_eng", "headset_med", "headset_sci", "headset_service", "headset_cargo")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1500)
+
+/datum/techweb_node/multy_keys
+	id = "multy_keys"
+	display_name = "Advanced Headset Keys"
+	description = "Headset keys that incrip more then one channle."
```
Comment (nik707) [inline code/modules/research/techweb/all_nodes.dm]: encrypt more than one channel.

**Review by nik707 — CHANGES_REQUESTED** [review]

**Comment by silicons** [comment]
I don't think most of these are something we want spam-built, sorry.

**Pull request closed without merging**